From 6aacc6b6bbe24d6f9f084507d9627838e297bd66 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E6=AE=B5=E5=BB=B7=E4=BC=9F?= <2624934035@qq.com>
Date: Mon, 11 May 2026 16:55:56 +0800
Subject: [PATCH] urma: fix ceqn upper-bound wording

The jfc ceqn field is validated with a strict less-than check against ceq_cnt, so update the public and test-facing comments to match the actual boundary.

Signed-off-by: 段廷伟 <2624934035@qq.com>
(cherry picked from commit 602a795ffefdc766469412288d71a2de38209b5f)
---
 src/urma/lib/urma/core/include/urma_cmd.h   | 4 ++--
 src/urma/lib/urma/core/include/urma_types.h | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/urma/lib/urma/core/include/urma_cmd.h b/src/urma/lib/urma/core/include/urma_cmd.h
index b3e7096a..e14cbe9a 100644
--- a/src/urma/lib/urma/core/include/urma_cmd.h
+++ b/src/urma/lib/urma/core/include/urma_cmd.h
@@ -556,7 +556,7 @@ typedef struct urma_cmd_create_jfc {
         uint32_t flag;
         int jfce_fd;
         uint64_t urma_jfc; /* urma jfc pointer */
-        uint32_t ceqn;     /* [Optional] event queue id, no greater than urma_device_cap_t->ceq_cnt
+        uint32_t ceqn;     /* [Optional] event queue id, less than urma_device_cap_t->ceq_cnt
                             * set to 0 by default */
     } in;
     struct {
@@ -605,7 +605,7 @@ typedef struct urma_cmd_alloc_jfc {
         uint32_t flag;
         int jfce_fd;
         uint64_t urma_jfc; /* urma jfc pointer */
-        uint32_t ceqn;     /* [Optional] event queue id, no greater than urma_device_cap_t->ceq_cnt
+        uint32_t ceqn;     /* [Optional] event queue id, less than urma_device_cap_t->ceq_cnt
                             * set to 0 by default */
     } in;
     struct {
diff --git a/src/urma/lib/urma/core/include/urma_types.h b/src/urma/lib/urma/core/include/urma_types.h
index 4c69d005..f7b41422 100644
--- a/src/urma/lib/urma/core/include/urma_types.h
+++ b/src/urma/lib/urma/core/include/urma_types.h
@@ -422,7 +422,7 @@ typedef union urma_jfc_flag {
 typedef struct urma_jfc_cfg {
     uint32_t depth;       /* [Required] the depth of jfc, no greater than urma_device_cap_t->jfc_depth */
     urma_jfc_flag_t flag; /* [Optional] see urma_jfc_flag_t, set flag.value to be 0 by default */
-    uint32_t ceqn;        /* [Optional] event queue id, no greater than urma_device_cap_t->ceq_cnt
+    uint32_t ceqn;        /* [Optional] event queue id, less than urma_device_cap_t->ceq_cnt
                               set to 0 by default */
     urma_jfce_t *jfce;    /* [Required] the event of jfc */
     uint64_t user_ctx;    /* [Optional] private data of jfc, set to NULL by default */
-- 
2.43.0
